src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -25,7 +25,7 @@ const AppealsV2StatusPage = ({ appeal }) => {
         description={currentStatus.description}/>
       <Alerts alerts={alerts}/>
       <WhatsNext nextEvents={nextEvents}/>
-      <Docket/>
+      <Docket {...docket}/>
```

Comment:
Is anything other than `total` and `ahead` passed along in the `docket` attribute?

Reply:
Yes actually. There's also `front`, `ready`, and `eta` which may or may not end up being used.